app/models/organisation.rb

```diff
@@ -138,6 +139,7 @@ def status
 
   def status_at(date)
     return :merged if merge_date.present? && merge_date < date
+    return :deactivated unless active
```

Comment:
Hmm, I feel like an org could be both of these at the same time (?) but should we/ do we allow merged organisations to get deactivated?

Reply:
Good question - I'll ask on the ticket.

Reply:
Looks like the answer is no - merged organisation should not be deactivatable.